modules/odc-customizing-available-cluster-roles-using-developer-perspective.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// applications/projects/working-with-projects.adoc
+
+[id="odc-customizing-available-cluster-roles-using-developer-perspective_{context}"]
+= Customizing the available cluster roles using the Developer perspective
+
+You can customize cluster role's name of existing projects by editing the YAML code in the `console.openshift.io/display-name: v1`.
```

Comment:
Some missing articles (see my rephrase). Also, what's the point of customizing the cluster role's name? See my rephrase to give a very quick, simple explanation of why users would care to customize the cluster role's name:

`Customize the cluster role's name of existing projects by editing the YAML code in the `console.openshift.io/display-name: v1`. When you customize the cluster role's name, you can <state benefit>.`

Reply:
The first part of the suggestion aligns with what is expected on the page. The other part is not required. So I'll rephrase accordingly.